RELEASE_NOTES.md

```diff
@@ -39,15 +33,9 @@ This document contains changes of oneTBB compared to the last release.
 - C++ exception handling mechanism on Windows* OS on ARM64* might corrupt memory if an exception is thrown from any oneTBB parallel algorithm (see Windows* OS on ARM64* compiler issue: https://developercommunity.visualstudio.com/t/ARM64-incorrect-stack-unwinding-for-alig/1544293).
 
 ## :hammer: Fixed Issues
-- Memory allocator crash on a system with an incomplete /proc/meminfo (GitHub* [#584](https://github.com/oneapi-src/oneTBB/issues/584)).
-- Incorrect blocking of task stealing (GitHub* #[478](https://github.com/oneapi-src/oneTBB/issues/478)).
-- Hang due to incorrect decrement of a limiter_node (GitHub* [#634](https://github.com/oneapi-src/oneTBB/issues/634)).
-- Memory corruption in some rare cases when passing big messages in a flow graph (GitHub* [#639](https://github.com/oneapi-src/oneTBB/issues/639)).
-- Possible deadlock in a throwable flow graph node with a lightweight policy. The lightweight policy is now ignored for functors that can throw exceptions (GitHub* [#420](https://github.com/oneapi-src/oneTBB/issues/420)).
-- Crash when obtaining a range from empty ordered and unordered containers (GitHub* [#641](https://github.com/oneapi-src/oneTBB/issues/641)).
-- Deadlock in a concurrent_vector resize() that could happen when the new size is less than the previous size (GitHub* [#733](https://github.com/oneapi-src/oneTBB/issues/733)).
+- Memory allocator crash when allocating ~1TB on 64-bit systems (GitHub* [#838](https://github.com/oneapi-src/oneTBB/issues/838)).
+- Thread Distribution over NUMA Nodes on Windows OS systems.
+- The function provided to oneapi::tbb::suspend is now guaranteed to be executed on the same thread.
```

Comment:
```suggestion
- The user function provided to oneapi::tbb::suspend is now guaranteed to be called on the same thread that called oneapi::tbb::suspend.
```

Reply:
May be it would be even better if we remove "`the same`":
`The user function provided to oneapi::tbb::suspend function is now guaranteed to be called on the thread that called oneapi::tbb::suspend`
What do you think?

Reply:
Maybe "For oneapi::tbb::suspend it is now guaranteed that the user-specified callable object is executed by the calling thread"
@pavelkumbrasev Is there a specification update patch that matches this change, or do we consider it an implementation specific behavior?